README.md

```diff
@@ -7,21 +7,32 @@
 
 (API) Rate limiting requests in CakePHP 3
 
-## Install
+## Requirements
 
-Using [Composer][composer]:
+- CakePHP 3.0+
+- CakePHP cache engine with support for atomic updates
+
+> Please note that this plugin will **not** work when using the default CakePHP
+> File Storage cache engine.
+
+## Installation
 
 ```
 composer require muffin/throttle:dev-master
 ```
+To make your application load the plugin either run:
+
+```
+Plugin::load('Muffin/Throttle');
```

Comment:
Above line says "..either run:" so this should be console command to load plugin